specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2019-10-27-preview/examples/OpenShiftManagedClustersGet.json

```diff
@@ -0,0 +1,82 @@
+{
+  "parameters": {
+    "api-version": "2019-10-27-preview",
+    "subscriptionId": "subid1",
+    "resourceGroupName": "rg1",
+    "resourceName": "clustername1"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/subid1/resourcegroups/rg1/providers/Microsoft.ContainerService/openShiftManagedClusters/clustername1",
+        "location": "location1",
+        "name": "clustername1",
+        "tags": {
+          "archv2": "",
+          "tier": "production"
+        },
+        "type": "Microsoft.ContainerService/OpenShiftManagedClusters",
+        "properties": {
+          "provisioningState": "Succeeded",
+          "openShiftVersion": "v3.11",
+          "clusterVersion": "osa.1",
+          "fqdn": "df9337748a4b.location1.cloudapp.azure.com",
+          "publicHostname": "openshift.1225bdf9337748a4bd3a.location1.azmosa.io",
+          "networkProfile": {
+            "vnetCidr": "10.0.0.0/8"
```

Comment:
VnetID is returned when the cluster is successfully deployed

Reply:
+ managementSubnetCIRD

Reply:
@julienstroheker this is what I have asked you before - in the other versions some fields do not appear anywhere in the examples, and you said to not worry about that. I've spend a lot of time trying to find the logic in what appears where and it didn't make much sense. That was the reason why I also removed the `managementSubnetCIRD` from my previous work (before we spoke in chat)

If it needs to be part of examples then we need to review other fields too. And possibly add them to the previous versions examples to make it consistent?

```
[ azure-rest-api-specs (aro-2019-20-27-preview) ] git grep -i  "\<vnetId\>" -- specification/containerservice/resource-manager/Microsoft.ContainerService
specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2019-09-30/openShiftManagedClusters.json:469:        "vnetId": {
specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2019-10-27-preview/openShiftManagedClusters.json:469:        "vnetId": {
specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-04-30/openShiftManagedClusters.json:469:        "vnetId": {

```

Reply:
Sorry for the confusion. 

Don't need to update the old one, let's make sure the `2019-10-27-preview` one are accurate.

Easy way to validate all the examples is running all the command locally (via curl) and validate the payloads.

Reply:
I agree with having more accurate examples. I will add missing bits.

The confusion was about the reason why this is missing from previous versions.

Reply:
I have added `vnetId` to the `specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2019-10-27-preview/examples/OpenShiftManagedClustersCreate_Update.json` can you please review this file and I will add it to other files?